.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode/launch.json
```

Comment:
You can grab a pre-made `.gitignore` from github (such as https://github.com/ONSdigital/git-diff-check/blob/master/.gitignore) then add in any extra bits you need